index.html

```diff
@@ -215,7 +214,50 @@ <h2 class="holder-title">
     <h3 class="last-title-container">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h3>
-    </div>
+    <form action="https://formspree.io/f/moqbyree" method="post">
+         <div class="form-block">
+              <div class="form-container">
+                <div class="input-name input-email-li">
+                    <input  class="input-email" type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
+                </div>
+                <div class="input-email-li">
+                    <input class="input-email" type="email" name="email" id="email" placeholder="Email address"
+                        required>
+                </div>
+                <div class="input-email-li">
+                    <textarea class="text-area" name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                        placeholder="Enter text here" required></textarea>
+                </div>
+                <div  class="submit-button input-email-li">
+                    <button class="form-btn" type="submit">
+                        Get in touch
+                    </button><br>
```

Comment:
- [OPTIONAL] According to the requirements you should Implement the button interactions (enable, hover, etc.). 💯

![image](https://user-images.githubusercontent.com/45231276/200808025-02fea14a-e54c-4c5e-8ad8-4c2c6842792f.png)